conda/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "zppy" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/E3SM-Project/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ab6dee537f8252a9f244f01b725046dd64748d68
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+
+  run:
+    - python >=3.6
```

Comment:
I think `zppy` has a bunch more dependencies that should explicitly be added.  Here's what I see so far (please keep alphabetized:
* `cdms2`
* `cdutil`
* `cdtime`
* `jinja2`
* `matplotlib-base`
* `netcdf4`
* `numpy`

I have a feeling this is an incomplete list building, installing and running the package on its own may catch others.

Reply:
As many as possible of these should also be added to the `install_requires` argument to `setup()` in `setup.py`.

Reply:
You just need to add:
```
    - configobj
    - jinja2
```